exports/defect_dojo.py

```diff
@@ -88,18 +100,30 @@ def _private_import(self, endpoint, data, filename):
             raise ValueError(f"Missing required entries for reimport: {missing}")
 
         if not self.token:
-            self.get_token()
+            if self._auth_and_set_token() == 1:
+                # failed to get token
+                return 1
 
         resp = requests.post(
             endpoint,
+            timeout=self.DD_CONNECT_TIMEOUT,
             headers=self.headers,
             data=data,
             files={"file": open(filename, "rb")},  # pylint: disable=consider-using-with
         )
         if resp.status_code >= 400:
-            print(vars(resp))
+            logging.debug(vars(resp))
```

Comment:
Should we also replace this `if resp.status_code >= 400:` with `try` + `resp.raise_for_status()` ? it looks much cleaner